### ISSUE TITLE
Gh-login does nothing

### ISSUE THREAD

**Issue by Kurt-Urban** [edgarjs/github-repos-alfred-workflow#46]
I put in gh-login *my-token* and press enter. Nothing happens after alfred closes the search bar and it doesnt log into my github. I don't believe I am doing anything incorrectly. I am currently on the macbook with the m1 pro, would the the arm architecture cause issues?

**Comment by edgarjs** [comment]
Thanks for reporting. It is a bug indeed. In the mean time you can visit [this URL] to generate your token.

**Comment by Kurt-Urban** [comment]
Is this token any different than the one that I generated using gh-token?

**Comment by edgarjs** [comment]
No, it should be the same. But don't you see this notification even?

![CleanShot 2022-02-22 at 17 59 23@2x](https://user-images.githubusercontent.com/12443/155240088-8840cf53-6024-4575-905d-da6fbb212125.png)

**Comment by edgarjs** [comment]
Ignore my first comment, I see you had a token already generated, but if it doesn't work still it may be the ruby version installed.

**Comment by Kurt-Urban** [comment]
> No, it should be the same. But don't you see this notification even?
> 
> ![CleanShot 2022-02-22 at 17 59 23@2x](https://user-images.githubusercontent.com/12443/155240088-8840cf53-6024-4575-905d-da6fbb212125.png)
> 
> 

It does not. It search bar closes and nothing else happens.

**Comment by Kurt-Urban** [comment]
> Ignore my first comment, I see you had a token already generated, but if it doesn't work still it may be the ruby version installed.

Oh, that may be the issue. I'm unsure if I even have ruby on my machine.

**Comment by Kurt-Urban** [comment]
I believe I only have the base version that comes installed on a mac. It may need updating.

**Comment by Kurt-Urban** [comment]
> Ignore my first comment, I see you had a token already generated, but if it doesn't work still it may be the ruby version installed.

![image](https://user-images.githubusercontent.com/80078706/155366564-beee6059-982d-447e-b3bb-ccd81beb0680.png)
Heres the logs

**Comment by b0lle** [comment]
are there any updates? I am running in the issues since I updated to alfred 5.

**Comment by edgarjs** [comment]
A new version of the workflow has been published. It uses the GitHub CLI under the hood, so authentication is handled by it now. Please make sure to follow the instructions from the README.md to install the requirements and authenticate properly. I'm closing this issue for now but, let me know if things break.
